CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# v2.0.0
+
+## New features
+- Plugin fully migrated to AiiDA 1.0. This version still supports both python 2.7 and python 3.x (>= 3.5) [[#32]](https://github.com/aiidateam/aiida-wannier90/pull/32) [[#34]](https://github.com/aiidateam/aiida-wannier90/pull/34) [[#63]](https://github.com/aiidateam/aiida-wannier90/pull/63) [[#64]](https://github.com/aiidateam/aiida-wannier90/pull/64) [[#66]](https://github.com/aiidateam/aiida-wannier90/pull/66)
+
+- Added parser for `.labelinfo.dat` file to get correct high-symmetry k-point labels in band_parser [[#88]](https://github.com/aiidateam/aiida-wannier90/pull/88)
+
+- Added support for the SCDM method [[#61]](https://github.com/aiidateam/aiida-wannier90/pull/61)
+
+## Fixes and general improvements
+
+- Implemented the `input_filename` and `output_filename` config options, replacing the `seedname` options. In this way, `verdi calcjob inputcat/outputcat` work as expected [[#42]](https://github.com/aiidateam/aiida-wannier90/pull/42)
+
+- Automatically retrieving all known output files generated by Wannier90 [[#79]](https://github.com/aiidateam/aiida-wannier90/pull/79) [[#80]](https://github.com/aiidateam/aiida-wannier90/pull/80)
+
+- Improved some parser key names [[#78]](https://github.com/aiidateam/aiida-wannier90/pull/78)
+
+- Parser: Add missing catch of "NotExistent" when kpoints_path is not given. [[#55]](https://github.com/aiidateam/aiida-wannier90/pull/55)
+
+- Removed opening transport in `prepare_for_submission`, avoiding the risk to open too many SSH connections [[#83]](https://github.com/aiidateam/aiida-wannier90/pull/83)
```

Comment:
Maybe we should mention that this means `remote_input_folder` and `local_input_folder` are mutually exclusive?

Reply:
Yes